File: src-tauri/tests/fixtures/symbol_index_languages/java/UserService.java
package com.example.users;

import java.util.Optional;
import static java.util.Collections.emptyList;

public final class UserService {
    public Optional<UserView> findUser(String id) {
        return Optional.empty();
    }
}

interface UserFormatter {
    String format(UserView user);
}

enum UserStatus {
    ACTIVE,
    DISABLED
}

record UserView(String id, String name) {
}

File: src-tauri/tests/fixtures/symbol_index_languages/csharp/UserService.cs
namespace Example.Users;

using System.Collections.Generic;
using static System.String;
using Json = System.Text.Json.JsonSerializer;

public sealed class UserService
{
    public UserDto FindUser(string id)
    {
        return new UserDto(id, Empty);
    }
}

public interface IUserFormatter
{
    string Format(UserDto user);
}

public enum UserStatus
{
    Active,
    Disabled
}

public record UserDto(string Id, string Name);
